### ISSUE TITLE
UI - Node management 

### ISSUE THREAD

**Issue by carlito-scality** [scality/metalk8s#905]
- [x] Design the workflow (https://docs.google.com/document/d/1n6R9NhE5u5JdYLcFzIt2U675Sdk0Avk-SdeWK6RDXqI/edit#)
- [x] Implement "Node creation" form to create a node via API Server (https://github.com/scality/metalk8s/issues/914)
- [ ] Implement "remove node"  via API Server (https://github.com/scality/metalk8s/issues/917) => On Hold
- [x] Make sur the deploy works correctly with Salt API
- [x] https://github.com/scality/metalk8s/issues/1044
- [x] https://github.com/scality/metalk8s/issues/1042
- [x] https://github.com/scality/metalk8s/issues/1045
-----------
Prerequisites: Salt API
- [x] Implement "Node expansion" part (install Salt, workload, add master role, add etcd role)
